app/scripts/metamask-controller.js

```diff
@@ -487,6 +494,12 @@ module.exports = class MetamaskController extends EventEmitter {
     await this.keyringController.submitPassword(password)
     const accounts = await this.keyringController.getAccounts()
 
+    // verify keyrings
+    const nonSimpleKeyrings = this.keyringController.keyrings.filter(keyring => keyring.type !== 'Simple Key Pair')
```

Comment:
Should we lowercase this string comparison? I lack context so maybe it's unnecessary, just a thought.

Reply:
here its ok, as it (should be) one of 2 hard coded values